src/core/app/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { AtomBinded } from '@reatom/core';
+export type { AppConfig } from './types';
+export const appRegistry = {
+  atoms: {},
+  listeners: {},
+};
+
+export function runAtom(atom: AtomBinded) {
```

Comment:
should that function be used instead of `store.dispatch` in several places in the code?
or should we consider the existence of that store?

Reply:
planned as forceRunAtom replacement